Makefile

```diff
@@ -25,9 +25,9 @@ path_build: ## Build the path binary locally (does not run anything)
 
 .PHONY: check_path_config
 check_path_config: ## Verify that path configuration file exists
-	@if [ ! -f ./local/path/config/.config.yaml ]; then \
+	@if [ ! -f ./local/path/.config.yaml ]; then \
 		echo "################################################################"; \
-   		echo "Error: Missing config file at ./local/path/config/.config.yaml"; \
+   		echo "Error: Missing config file at ./local/path/.config.yaml"; \
```

Comment:
Does this need to be changed in this PR (or at all)?

Reply:
Yes this is part the reorganization of the local file to make it clearer. Keeping the configuration files at the same level is IMO important to reduce unnecessary complexity and depth in the folder structure. tl;dr it's easier to keep the local config files organized.

Reply:
Not opposed to it long term (ideally with a CLI flag): but this introduces more changes and will break previously working tests / environments.